src/controllers/AuthController.js

```diff
@@ -0,0 +1,68 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const logErrorHelper = require('../logger/loggerHelper');
+const { secret } = require('../../config/jwtConfig').jwtData;
+const authHelper = require('../helpers/authHelper');
+export default class AuthController {
+    constructor(tokenList, usersList) {
+        this.tokenList = tokenList;
+        this.usersList = usersList;
+    }
+
+    async login(req, res, next) {
+        try {
+            const user = await this.usersList.findOne({
+                where: {
+                    login: req.body.login
+                },
+                raw: true
+            });
+
+            if (!user) {
+                return res.status(404).json(`${req.body.login} does not exist. Please register first!`);
+            }
+
+            if (bcrypt.compareSync(req.body.password, user.password)) {
```

Comment:
@lenon777, the AuthController should know nothing about bcrypt. Extract work with bcrypt to separate service and inject it via controller constructor.